samps/contests/interactive/config/system.yaml

```diff
@@ -0,0 +1,65 @@
+clar-categories:
+- judges
+- sysops
+- operations
+- ccs
+default-clars:
+- Read the problem statement carefully
+- No comment
+languages:
+  - name: 'Java'
+    compilerCmd: '/usr/bin/javac -encoding UTF-8 -sourcepath . -d . {files}'
```

Comment:
The use of paths like `/usr/bin/javac` makes this "sample contest problem" configuration _platform-specific_.  I would strongly prefer seeing all of our sample contests use platform-independent (e.g. relative) paths -- that is, assume that things like `javac` are in the PATH, so that the files will work on any platform.

Note that this comment also applies to numerous lines below; I won't bother to add separate comments on each of those...

Reply:
Since interactive problems only work with Linux (at the moment), having Linux paths is not a major issue; at least that was my thought process.  I can just remove the FQP's and it SHOULD still work on Linux.  WIll try it.

Reply:
Again, I based my comment on the concept that our sample contests should not be platform-specific, just like I believe they should not (normally) override defaults.  I think it would be better if the FQP's were made "non-qualified" (i.e. so they rely on the PATH, which is something we tell users right at the beginning of the Admin Guide that they must set -- at least for Java (and I think for all languages...)